### PR TITLE
Fixed - implict declaration - 4.19-rc2

### PR DESCRIPTION
**Pull request by alifiratari**
get_monotonic_boottime

### CODE REVIEW

File: r6p2/src/devicedrv/mali/linux/mali_osk_time.c [hunk 1]
```diff
@@ -54,6 +54,6 @@ u64 _mali_osk_time_get_ns(void)
 u64 _mali_osk_boot_time_get_ns(void)
 {
 	struct timespec tsval;
-	get_monotonic_boottime(&tsval);
+	getboottime(&tsval);
```
Comment (mripard) [inline r6p2/src/devicedrv/mali/linux/mali_osk_time.c]: What is it fixing exactly? On which kernel, with which version?

Comment (alifiratari) [reply r6p2/src/devicedrv/mali/linux/mali_osk_time.c]: for 4.19-rc2 kernel, its for our build (Orange Pi Win Plus A64 )

Comment (alifiratari) [reply r6p2/src/devicedrv/mali/linux/mali_osk_time.c]: get_monotonic_boottime has been disabled, in this patch https://lore.kernel.org/patchwork/patch/978060/

**Comment by mripard** [comment]
This already builds. Closing the PR

**Pull request closed without merging**